### PR TITLE
feat(workload): add daemonset serializer

### PR DESCRIPTION
**Pull request by whalecold**
**What this PR does / why we need it**:

feat(workload): add daemonset serializer

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by whalecold** [comment]
@zxq-bit

File: serializer/apps/v1/daemonset.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/caicloud/rudder-client/serializer/universal"
```
Comment (zxq-bit) [inline serializer/apps/v1/daemonset.go:7]: 似乎应该单独作为第三个组
https://github.com/caicloud/engineering/blob/master/docs/golang_conventions.md#order

**Pull request merged**